ddtrace/tracer/post_processor.go

```diff
@@ -0,0 +1,111 @@
+package tracer
```

Comment:
Make sure to copy-pasta the copyright headers.

Reply:
Ah, forgot that 😞. Thanks!